src/App.css

```diff
@@ -1,5 +1,7 @@
 .App {
   text-align: center;
+  margin-top: 40px;
+  margin-bottom: 80px;
```

Comment:
use the shorthand notation for margins:

Like this: `margin: 40px 0 80px 0` also explore more shorthand like this `margin: 2px 4px`